app/client/ui/AdminPanel.ts

```diff
@@ -420,12 +441,92 @@ isolated from other documents and isolated from the network.'),
       )
     });
   }
+
+  /**
+   * Show the results of various checks. Of the checks, some are considered
+   * "redundant" (already covered elsewhere in the Admin Panel) and the
+   * remainder are "novel".
+   */
+  private _buildProbeItems(owner: MultiHolder, options: {
+    showRedundant: boolean,
+    showNovel: boolean,
+  }) {
+    return dom.domComputed(
+      use => [
+        ...use(this._checks.probes).map(probe => {
+          const isRedundant = probe.id === 'sandboxing';
+          const show = isRedundant ? options.showRedundant : options.showNovel;
+          if (!show) { return null; }
+          const req = this._checks.requestCheck(probe);
+          return this._buildProbeItem(owner, req.probe, use(req.result), req.details);
+        }),
+      ]
+    );
+  }
+
+  /**
+   * Show the result of an individual check.
+   */
+  private _buildProbeItem(owner: MultiHolder,
+                          info: BootProbeInfo,
+                          result: BootProbeResult,
+                          details: ProbeDetails|undefined) {
+
+    const status = (result.success !== undefined) ?
+        (result.success ? '✅' : '❗') : '―';
+
+    return this._buildItem(owner, {
+      id: `probe-${info.id}`,
+      name: info.id,
+      description: info.name,
+      value: cssStatus(status),
+      expandedContent: [
+        cssCheckHeader(
+          'Results',
+          { style: 'margin-top: 0px; padding-top: 0px;' },
+        ),
+        result.verdict ? dom('pre', result.verdict) : null,
+        (result.success === undefined) ? null :
+            dom('p',
+                result.success ? 'Check succeeded.' : 'Check failed.'),
+        (result.done !== true) ? null :
+            dom('p', 'No fault detected.'),
+        (details?.info === undefined) ? null : [
+          cssCheckHeader('Notes'),
+          details.info,
+        ],
+        (result.details === undefined) ? null : [
+          cssCheckHeader('Details'),
+          ...Object.entries(result.details).map(([key, val]) => {
+            return dom(
+              'div',
+              cssLabel(key),
+              dom('input', dom.prop(
+                'value',
+                typeof val === 'string' ? val : JSON.stringify(val))));
+          }),
+        ],
+      ],
```

Comment:
Do these need translating (Where we add text to the DOM) with `t()`? Both string literals and the constant strings coming out of `details`.

Reply:
I'm not sure if `info` from the probe result is localized by the backend, but it's probably out of scope for this task either way I'd think!